src/common/session.ts

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
```

Comment:
Context is kind of special and different from other files in this folder.  So should we create "src/context" folder and move this there?